src/ui/view/governance/details/details.tsx

```diff
@@ -78,12 +79,16 @@ const Article: FC<ArticleProps> = ({ article, isOnlyArticle }) => {
   const theme = useAppStore(store => store.theme)
   const isFirstArticle = article.number.endsWith('1')
   const [isOpen, setIsOpen] = useState<boolean>(isFirstArticle)
-
   const toggleArticle = useCallback(() => setIsOpen(!isOpen), [isOpen])
 
   return (
     <div className="okp4-dataverse-portal-governance-details-article-container">
-      <div className="okp4-dataverse-portal-governance-details-article-header">
+      <div
+        className={classNames('okp4-dataverse-portal-governance-details-article-header', {
+          'pointer-cursor': !isOnlyArticle
+        })}
+        onClick={!isOnlyArticle ? toggleArticle : undefined}
```

Comment:
A more idiomatic approach might be to handle this conditional logic inside an `handleClick` function.
Example:
```tsx
const handleClick = () => {
  if (!isOnlyArticle) {
    toggleArticle();
  }
}
// ...
onClick={handleClick}
```

Reply:
Yes you're right better to have a dedicated handler